sdk/core/azure-core/src/main/java/com/azure/core/util/HttpClientOptions.java

```diff
@@ -191,13 +212,76 @@ public HttpClientOptions setReadTimeout(Duration readTimeout) {
 
     /**
      * Gets the read timeout duration used when reading the server response.
+     * <p>
+     * By default the read timeout is 60 seconds.
      *
      * @return The read timeout duration.
      */
     public Duration getReadTimeout() {
         return getTimeout(readTimeout);
     }
 
+    /**
+     * Sets the maximum connection pool size used by the underlying HTTP client.
+     * <p>
+     * By default the maximum connection pool size is determined by the underlying HTTP client.
```

Comment:
We should also note that this is not a global max. It is the max pool size for that specific client.

Reply:
I'll update this